libs/components/forms/src/lib/modules/input-box/input-box.component.scss

```diff
@@ -119,6 +119,7 @@ sky-input-box {
   .sky-input-box-hint-text {
     flex-basis: 100%;
     margin-top: var(--sky-margin-stacked-xs);
+    text-align: left;
```

Comment:
The file drop component adds `text-align: center` to its containers; this addition makes sure all instances of input box's hint text are aligned "left".